.cirrus.yml

```diff
@@ -292,22 +277,25 @@ special_testing_task:
 # necessary to execute them within a PR to validate changes.
 optional_testing_task:
 
+    depends_on:
+        - "gating"
+
     # Only run system tests in PRs (not on merge) if magic string is present
     # in the PR description.  Post-merge system testing is assumed to happen
     # later from OS distribution's build systems.
     only_if: >-
         $CIRRUS_BRANCH != 'master' &&
+        $CIRRUS_CHANGE_MESSAGE !=~ '.*\*\*\*\s*CIRRUS:\s*TEST\s*IMAGES\s*\*\*\*.*' &&
         $CIRRUS_CHANGE_MESSAGE =~ '.*\*\*\*\s*CIRRUS:\s*SYSTEM\s*TEST\s*\*\*\*.*'
 
     gce_instance:
-        image_project: "libpod-218412"
         matrix:
             image_name: "${FEDORA_CACHE_IMAGE_NAME}"
             image_name: "${PRIOR_FEDORA_CACHE_IMAGE_NAME}"
             image_name: "${UBUNTU_CACHE_IMAGE_NAME}"
-            # TODO: Make these work (also testing_task above)
+            image_name: "${PRIOR_RHEL_CACHE_IMAGE_NAME}"
             # image_name: "${RHEL_CACHE_IMAGE_NAME}"
-            # image_name: "${PRIOR_RHEL_CACHE_IMAGE_NAME}"
+            image_name: "${PRIOR_CENTOS_CACHE_IMAGE_NAME}"
```

Comment:
PRIOR_CENTOS_CACHE_IMAGE_NAME does not seem to be defined. Will that cause problems here? (There's another instance of it being referenced, but it's in a context where I think undef will be handled fine)

Reply:
ATM those are only used from w/in ``.cirrus.yml`` though CentOS and RHEL are fairly defunct and I should probably just strip them all out.  With so many dependencies, testing ends up needing a franken-distro to stand a chance of success...with frequent breakage and guaranteed dis-use (of franken-distro), it's just better we leave those as downstream consumers (for testing purposes).